### ISSUE TITLE
1.4.0: Mappings no longer affecting IntelliSense suggestions

### ISSUE THREAD

**Issue by mre00** [ChristianKohler/PathIntellisense#68]
After updating to 1.4.0, the mappings no longer appear to be working correctly.  My mappings setting:

`    "path-intellisense.mappings": {
        "~": "${workspaceRoot}/src"
    }
`

When typing '~/', the IntelliSense suggestions are relative to the current file's directory and not the <root>/src directory.

**Comment by lekhnath** [comment]
Same here I just upgraded to v1.4.0 and everything breaks. I've following mappings.
```
"path-intellisense.mappings": {
    "lib": "${workspaceRoot}/server/lib",
    "app": "${workspaceRoot}/src/app"
  }
```

**Comment by ChristianKohler** [comment]
There was a bug in 1.4.0. Could you update to 1.4.2 and check whether this fixes this issue?

**Comment by lekhnath** [comment]
@ChristianKohler  It seems the issue is fixed with the latest v1.4.2 release. Thanks.

**Comment by ChristianKohler** [comment]
Great. Thank you @lekhnath for the update.
